src/index.ts

```diff
@@ -64,12 +91,11 @@ export const sakiewkaModule = (currency: Currency, btcNetwork: string): Sakiewka
     transaction: transactionModule,
     address: addressModule,
     key: keyModule,
-    bitcoin: bitcoinOperations,
```

Comment:
manual czy automat? Bo jak manual to ja bym zostawił, trailing comma sa spoko

Reply:
IntelliJ